docs/data/material/components/tabs/NavTabs.tsx

```diff
@@ -24,7 +43,13 @@ export default function NavTabs() {
   const [value, setValue] = React.useState(0);
 
   const handleChange = (event: React.SyntheticEvent, newValue: number) => {
-    setValue(newValue);
+    // event.type can be equal to focus with selectionFollowsFocus.
+    if (
+      event.type !== 'click' ||
+      (event.type === 'click' && samePageLinkNavigation(event as any))
```

Comment:
To avoid `any`?
```suggestion
      (event.type === 'click' &&
        samePageLinkNavigation(
          event as React.MouseEvent<HTMLAnchorElement, MouseEvent>,
        ))
```